src/Status/StatusCountBar/StatusCountBar.stories.tsx

```diff
@@ -0,0 +1,101 @@
+import { Meta, StoryFn } from "@storybook/react";
+
+import { Box } from "@mui/material";
+import React from "react";
+import StatusCountBar from "./StatusCountBar";
+import { StatusCountBarProps } from "./StatusCountBar.types";
+
+/**
+ * Story metadata
+ */
+const meta: Meta<typeof StatusCountBar> = {
+  component: StatusCountBar,
+  title: "Status/StatusCountBar"
+};
+export default meta;
+
+/**
+ * Story template for the StatusCountBar component
+ */
+const Template: StoryFn<StatusCountBarProps> = args => {
+  return <StatusCountBar {...args} />;
+};
+
+/**
+ * Default story for the StatusCountBar component
+ */
+/* eslint-disable sort-keys */
```

Comment:
I have added this because the order of the count object we pass is the order in the figma that should be kept, and the linter is not allowing us the desired order.

Reply:
We have discussed this with @evoinea and I have added additional comment above each eslint disable comment for the sort keys.

Reply:
https://github.com/IPG-Automotive-UK/react-ui/pull/955#issuecomment-2374375511